src/js/components/user-management/userlist.js

```diff
@@ -46,7 +46,9 @@ const UserList = ({ currentUser, editUser, isEnterprise, removeUser, roles, user
               <TableCell>
                 <RelativeTime updateTime={user.updated_ts} />
               </TableCell>
-              {isEnterprise && <TableCell>{(user.roles || []).map(role => roles.find(currentRole => currentRole.id === role).title).join(', ')}</TableCell>}
+              {isEnterprise && (
+                <TableCell>{(user.roles || []).map(role => (roles.find(currentRole => currentRole.id === role) || {}).title).join(', ')}</TableCell>
```

Comment:
lovely!
that will print an empty string followed by `,` in case there are unrecognised groups.